scripts/content-repo-preview/build.ts

```diff
@@ -0,0 +1,95 @@
+import { execFileSync } from 'child_process'
+import path from 'path'
+import fs from 'fs'
+
+async function main() {
+  // TODO: check for necessary env vars?
```

Comment:
Would the idea here be to throw an error if any of the required env vars aren't set? I'd be on board with that! (Would also be on board with merging this as-is, if we want).

Reply:
Yeah that was my thought! We need REPO, DEV_IO and IS_CONTENT_PREVIEW, right?